### DISCUSSION TITLE
"Cannot determine transform between probe and phantom (ProbeToPhantom transform is invalid)"

### DISCUSSION THREAD

**Discussion by NesBarn** [PlusToolkit/PlusLib#953] [General]
Hello,

When I perform the spatial calibration to get the image to Probe transform, I always have the message in orange appearing on the left saying: "cannot determine transform between probe and phantom (ProbeToPhantom transform is invalid). Make sure all markers are in the field of view". Naturally I checked all the markers are visible, which is the case plus I did a test where I let the probe stationary to be sure all markers are visible.
Any idea why this happens?
Thanks in advance

**Comment by Sunderlandkyl** [comment]
Can you upload your config file and a log?
